src/main/kotlin/no/nav/syfo/dkif/DigitalKontaktinfo.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.syfo.dkif
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+
+data class DigitalKontaktinfo (
+    val kanVarsles: Boolean?,
+    val reservert: Boolean?,
```

Comment:
Må disse være optional? Er jo endel lettere å jobbe med hvis de ikke er det

Reply:
@kaja-nav Det virker som de er nullable på `digdir-krr-proxy` sin side, så da tror jeg vi må ta høyde for det.

Reply:
Men hva betyr det hvis de er null?

Reply:
Hvis det er det samme som false så kan vi kanskje sette dem til false dersom de er null? Eller tenker du at det er problematisk?

Reply:
@kaja-nav Ja, det kan gjøres 👍

Reply:
@kaja-nav Det skjer vel egentlig implisitt her:
https://github.com/navikt/syfooppfolgingsplanservice/pull/382/files#diff-c2580460840d700f6bd62751a34cf262f9fda75bc8323ca1d382af030bd9466bR68
`reservert` var også nullable i de "gamle" DKIF-kallene, så nå gjøres det samme for `kanVarsles`

Reply:
Ja, jeg tenkte bare at det var lettere å håndtere de hvis de ikke var nullable, men det er ikke noe must

Reply:
@kaja-nav Ser poenget ditt 👍 Fikset